Remotely.Tests.ps1

```diff
@@ -131,9 +131,49 @@
             Remove-Item $configFile -ErrorAction SilentlyContinue -Force
         }
     }
-    
+}
+
+Describe "ConfigurationName" {
+    $script:configFile = (join-path $PSScriptRoot 'machineConfig.csv')
```

Comment:
> $script:configFile = (join-path $PSScriptRoot 'machineConfig.csv') [](start = 3, length = 67)

Can you add this to BeforeAll. $configfile does not need to be script variable, defining it in BeforeAll should be sufficient.